README.md

```diff
@@ -39,6 +39,8 @@ your will.
 There is no set purpose.  What ChaosBot makes itself into is entirely up to
 the imagination of the open source community.
 
+Wondering how to contribute? Try adding a feature requested [here](https://github.com/chaosbot/chaos/wiki/Feature-Requests).
```

Comment:
English is not my first language, but how can a link request a feature?

I would think that "a user would request a feature", or that "a user would add a feature request", not that "a user would add a feature requested", because "requested" makes me wonder who requests the feature that the user is about to add.

Perhaps a native speaker could chime in here.